sdk/core/azure-core-amqp/CHANGELOG.md

```diff
@@ -3,13 +3,22 @@
 ## 2.1.0-beta.1 (Unreleased)
 
 ### New Features
+
 - Exposing CbsAuthorizationType.
 - Exposing ManagementNode that can perform management and metadata operations on an AMQP message broker.
 - AmqpConnection, AmqpSession, AmqpSendLink, and AmqpReceiveLink extend from AsyncCloseable.
 - Delivery outcomes and delivery states are added.
 
 ### Bug Fixes
+
 - Fixed a bug where connection and sessions would not be disposed when their endpoint closed.
+- Fixed a bug where ReactorExecutor did not dispose of its scheduler when "IO Sink was interrupted".
+
+### Dependency Updates
+
+- Upgraded `azure-core` from `1.15.0` to `1.16.0`.
```

Comment:
This should be upgrading to `1.17.0` right? That was the version just released.

Reply:
Oh yeah. Goot catch